### PR TITLE
fix: firefox test

### PR DESCRIPTION
**Pull request by wadjih-bencheikh18**
closes https://github.com/cheminfo/nmrium/issues/2255

### CODE REVIEW

File: test-e2e/panels/filters.test.ts [hunk 1]
```diff
@@ -47,7 +47,9 @@ async function baselineCorrectionFilter(
   } else {
     await nmrium.clickTool('baselineCorrection');
   }
-  await nmrium.page.click('button >> text=Apply');
+  // wait for BaseLineCorrectionPanel
+  await nmrium.page.waitForSelector('_react=BaseLineCorrectionPanel');
+  await nmrium.page.click('_react=BaseLineCorrectionPanel >> text=Apply');
```
[the remark this answers is not in the document]
Comment (wadjih-bencheikh18) [reply test-e2e/panels/filters.test.ts:52]: Thank you, I'm trying to figure out the problem

**Comment by wadjih-bencheikh18** [comment]
Click event was being intercepted by a `<dialog>` element in the center of apply `button`.I fixed the test by changing click position .I couldn't figure out why there is `<dialog>` element

File: test-e2e/panels/filters.test.ts [hunk 2]
```diff
@@ -47,7 +47,9 @@ async function baselineCorrectionFilter(
   } else {
     await nmrium.clickTool('baselineCorrection');
   }
-  await nmrium.page.click('button >> text=Apply');
+  await nmrium.page.click('button >> text=Apply', {
```
Comment (targos) [inline test-e2e/panels/filters.test.ts:50]: Can you at least add a `// TODO: explanation` comment? We shouldn't push workarounds without minimal documentation.

Comment (wadjih-bencheikh18) [reply test-e2e/panels/filters.test.ts:51]: added, thank you

**Comment by hamed-musallam** [comment]
@wadjih-bencheikh18 

the issue happens because the dialog is taking space in the layout which should not if no nodes inside it, this happens because of the `display:flex` which i think no need for it  https://github.com/zakodium-oss/react-science/pull/462

**Pull request closed without merging**